pycheribuild/projects/disk_image.py

```diff
@@ -1226,6 +1226,9 @@ def add_required_libraries(self, libdirs: "list[str]", ignore_required: bool = F
         ]
         # required, but only recently added so not always present
         optional_libs += ["libsys.so.7"]
+        # required for libcrypto.so.5 versions newer than CheriBSD
+        # commit 4ca6f578846d947386821de5e740062c83f8376a
```

Comment:
Can you add to the comment mentioning the soname bump to 7 being recent hence why it’s optional? (The fact libcrypto optionally needs it isn’t a reason to put it in optional, because if it exists then it’ll be included)